src/msg/async/EventKqueue.cc

```diff
@@ -156,8 +156,13 @@ int KqueueDriver::add_event(int fd, int cur_mask, int add_mask)
     }
   }
   // keep what we set
-  if (fd >= sav_max)
-    resize_events(sav_max+5000);
+  if (fd >= sav_max) {
+    r = resize_events(sav_max+5000);
+    if (r < 0) {
+      lderr(cct) << __func__ << " event count is exceed." << dendl;
+      return r;
```

Comment:
@tchaikov I am not quite clear about the logic behind this now. Should we keep silent if resize_events() fails here.
I saw error code being returned in add_events()

  int r = test_thread_change(__func__);
  if ( r < 0 )
    return r;
Shall we deal with the error from resize_events in the same way?